src/psycopmlutils/loaders/load_outcomes.py

```diff
@@ -38,3 +38,40 @@ def any_diabetes():
         msg.good("Finished loading any_diabetes event times")
         output = df[["dw_ek_borger", "timestamp", "value"]]
         return output.reset_index(drop=True)
+
+    @data_loaders.register("forced_admissions_indlagt")
+    def forced_admnissions_indlagt():
+
+        df = sql_load(
+            "SELECT dw_ek_borger, timestamp FROM [fct].[psycop_fa_outcomes_all_disorders_tvangsindlaeg_Indlagt_2y_0f_2015-2021]",
```

Comment:
`SELECT X, Y, Z` means "select the X, Y and Z columns from the SQL table. If Y doesn't exist, it isn't selected. Since you rename `datotid_slut` to `timestamp` below, I assume `psycop_fa_outcomes_all_disorders_tvangsindlaeg_Indlagt_2y_0f_2015` doesn't contain a `timestamp` column?

Reply:
Thanks, overlooked this mistake